setMarginBankWithdrawal.js

```diff
@@ -0,0 +1,35 @@
+const { Guardian__factory } = require("@firefly-exchange/library/dist/src/contracts/exchange");
+const { ethers } = require("@firefly-exchange/library");
+
+const environment = require("./environments.json");
+require('dotenv').config();
+env = environment[process.env.env]
+market = process.env.market;
+
+console.log("Env:", process.env.env);
+console.log("Market:", market);
+
+
+const providerURL = env["URL"];
+const guardian = env["DEPLOYER"]
+const marginBankContractAddress = env[market]["MarginBank"]
+const guardianContractAddress = env[market]["Guardian"]
+
+const provider = new ethers.providers.JsonRpcProvider(providerURL);
+const guardianWallet = new ethers.Wallet(guardian, provider);
+
+async function main(moveTo){
+    const destination = moveTo == 1 ? "off-chain" : "on-chain"
```

Comment:
margin bank withdrawl is enabled or disabled, not moved to on-chain or off-chain, please update strings